File: Baekjoon/Level2/2-03/main.go
package main

import "fmt"

func main() {
    var year int
    fmt.Scan(&year)

    if (year%4 == 0 && year%100 != 0) || year%400 == 0 {
        fmt.Println(1)
    } else {
        fmt.Println(0)
    }
}

File: Baekjoon/Level2/2-03/main.dart
import 'dart:io';

int solution(int year) {
  int answer = 0;
  if ((year % 4 == 0 && year % 100 != 0) || year % 400 == 0) {
    answer = 1;
  }
  return answer;
}

void main() {
  int year = int.parse(stdin.readLineSync()!);

  print(solution(year));
}
